src/robots/msj_platform.cpp

```diff
@@ -263,11 +279,22 @@ int main(int argc, char *argv[]) {
     }
     ROS_INFO("\nurdf file path: %s\ncardsflow_xml %s", urdf.c_str(), cardsflow_xml.c_str());
 
-    MsjPlatform robot(urdf, cardsflow_xml);
+    int workers = atoi( argv[1]);
+    cout << "\nNUMBER OF WORKERS " << workers << endl;
+
+    ///Multiple instances of workers are created for parallelized training.
+    MsjPlatform **ptr = NULL;
```

Comment:
Is it not usual to use a list or some data structure?

Reply:
maybe vector. But how can I inject distinct ids to each element of the vector?

Reply:
Now it is with vector of shared_ptrs